target/linux/ath79/dts/qca9563_tplink_eap2x5_1port.dtsi

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
Also for DTSI files, we typically used hyphens for the "model part", i.e.
qca9563_tplink_eap2x5-1port.dtsi

Reply:
I moved the DTSI and did a quick build test to make sure I didn't miss renaming it in the DTS-files.